src/modules/app/route-configs/SprungbrettRouteConfig.js

```diff
@@ -23,24 +23,35 @@ type RequiredPayloadsType = {|sprungbrettJobs: Payload<Array<SprungbrettModel>>,
 export const SPRUNGBRETT_ROUTE = 'SPRUNGBRETT'
 export const SPRUNGBRETT_EXTRA = 'sprungbrett'
 
+const fetchExtras = async (dispatch, getState) => {
+  const state = getState()
+  const {city, language} = state.location.payload
+  const extrasPayload = await fetchData(extrasEndpoint, dispatch, state.extras, {city, language})
+  const extras: ?Array<ExtraModel> = extrasPayload.data
+
+  if (extras) {
+    const sprungbrettExtra: ExtraModel | void = extras.find(extra => extra.alias === SPRUNGBRETT_EXTRA)
+    if (sprungbrettExtra) {
+      const params = {city, language, url: sprungbrettExtra.path}
+      const sprungbrettEndpoint1 = sprungbrettEndpoint
```

Comment:
Why do we need this variable?

Reply:
Honestly I don't know. Just copied this.